weave/trace/vals.py

```diff
@@ -145,6 +146,12 @@ def attribute_access_result(
 ) -> Any:
     # Not ideal, what about properties?
     if callable(val_attr_val):
+        sig = inspect.signature(val_attr_val)
```

Comment:
I think this code should be a function in op.py called like bind_op()

Reply:
clarified as `maybe_bind_method` in `op.py`